data/ecosystem/distributions.yaml

```diff
@@ -0,0 +1,36 @@
+# cSpell:ignore Splunk AWS ADOT Grafana observIQ BindPlane RHOSDT RedHat Sumo Liatrio
+- name: Splunk Distribution of OpenTelemetry Collector
+  url: https://github.com/signalfx/splunk-otel-collector
+  docsUrl: https://docs.splunk.com/observability/en/gdi/opentelemetry/opentelemetry.html
+  components: [Collector]
+  oss: true
+- name: AWS Distribution for OpenTelemetry Collector (ADOT Collector)
```

Comment:
AWS Distro for OpenTelemetry (ADOT)

Reply:
@mhausenblas Feel free to edit my branch directly!